xlive/H2MOD/Modules/OnScreenDebug/OnscreenDebug.cpp

```diff
@@ -49,7 +48,7 @@ void addDebugText(char* text) {
 	strncpy(DebugStr[DebugTextArrayPos], text, lenInput);
 	memset(DebugStr[DebugTextArrayPos] + lenInput, 0, 1);
 
-	onscreendebuglog->debug(text);
+	onscreendebug_log->debug(text);
```

Comment:
no underscores, camelcase for variable names (yes I know its not consistent across the codebase)